src/shell/dotnet/examples/module-catalog.json

```diff
@@ -34,5 +34,22 @@
             "Url": "https://fdc3.finos.org/toolbox/fdc3-workbench/",
             "IconUrl": "https://fdc3.finos.org/toolbox/fdc3-workbench/favicon.ico"
         }
-    }
+    },
+	{
+		"Id": "Datagrid Example",
+        "Name": "Js Datagrid Example",
+        "ModuleType": "web",
+        "Details": {
+            "Url": "http://localhost:4200/",
+			"IconUrl": "https://brandslogos.com/wp-content/uploads/images/large/angular-icon-logo.png"
```

Comment:
Developers would just use the favicon here, similar to what you see on line 35.
I checked the favicon is there for the project so please just set that here.
It's available from the url path like "http://localhost:4200/favicon.ico"

However I like the idea of showcasing "COMPOSE_ALLOWED_IMAGE_SOURCES", but from at the moment what we have available the chart or one of the simple url loads would be better candidates.
If you'd like to do an example for this on this PR I'd suggest to go with the chart and use an icon that's under the creative commons license like e.g: https://commons.wikimedia.org/wiki/File:Simpleicons_Business_bars-chart-ascending.svg.

But it would be better to just create a very simple example that showcases only this feature on a separate PR.